CHANGELOG

```diff
@@ -1,4 +1,5 @@
 next
+* Support for a --quiet option to suppress tools output. Many tools converted to use it.
```

Comment:
to suppress tool output to stdout...

Reply:
@williamcroberts right, I added more information to the changelog entry.